builtin/logical/pki/backend_test.go

```diff
@@ -0,0 +1,440 @@
+package pki
+
+import (
+	"crypto"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"math"
+	"math/rand"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/fatih/structs"
```

Comment:
Any chance we can remove this dep? Looks like its not used for much but forces so much additional code to be pulled in